src/Directory.Build.props

```diff
@@ -1,5 +1,356 @@
 <Project>
-  <Import Project="..\Directory.Build.props" />
+  <!-- Base repo layout properties -->
```

Comment:
I take it all of this here and in d.b.targets is a copy-paste of what was in the root?

Reply:
correct.